apps/condo/domains/contact/components/ContactsEditor.tsx

```diff
@@ -0,0 +1,556 @@
+                                            left={AnotherContactLabel}
+                                        />
+                                        <ContactSyncedAutocompleteFields
+                                            initialValue={initialValue || manuallyTypedContact}
+                                            onChange={handleChangeContact}
+                                            onChecked={handleSyncedFieldsChecked}
+                                            checked={editableFieldsChecked}
+                                            contacts={contacts}
+                                            displayMinusButton={true}
+                                            onClickMinusButton={handleClickOnMinusButton}
+                                        />
+                                        {!role.canManageContacts && (
+                                            <Col span={24}>
+                                                <ErrorsWrapper>
+                                                    {CannotCreateContactMessage}
+                                                </ErrorsWrapper>
+                                            </Col>
+                                        )}
+                                    </>
+                                ) : (
+                                    <Col span={24}>
+                                        <Button
+                                            type="link"
+                                            style={{
+                                                color: green[6],
+                                                paddingLeft: '5px',
+                                            }}
+                                            onClick={handleClickOnPlusButton}
+                                            icon={<PlusCircleFilled style={{ color: green[6], fontSize: 21, position: 'relative', top: '2px' }}/>}
+                                        >
+                                            {AddNewContactLabel}
+                                        </Button>
+                                    </Col>
+                                )}
+                            </>
+                        </>
+                    )}
+                </Row>
+                {/*
+                    This is a place for items of external form, this component is embedded into.
+                    Why not to use them in place of actual inputs?
+                    Because we have many inputs ;)
+                    1. Input pairs, imitating radio group for select
+                    2. Text inputs for manual typing
+                    Logic of displaying `Form.Item`, depending on what is currently selected:
+                    radio-like pair, or manual input pair, — will be complex.
+                    The simplest solution, i currently know, — is to keep it in one place.
+                    So, we use hidden inputs here, but reveal validation errors.
```

Comment:
Redurant any comment

Reply:
I don't think so, because I've clearly described, why I'm using this approach.